lib/mpool/src/mblock_file.c

```diff
@@ -952,13 +970,10 @@ mblock_file_delete(struct mblock_file *mbfp, uint64_t *mbidv, int mbidc)
     if (err)
         return err;
 
-    mblocksz = mbfp->mblocksz;
     /* Discard mblock */
-    rc = fallocate(
-        mbfp->fd,
-        FALLOC_FL_PUNCH_HOLE | FALLOC_FL_KEEP_SIZE,
-        block_off(*mbidv, mblocksz),
-        mblocksz);
+    mblocksz = mbfp->mblocksz;
+    rc = fallocate(mbfp->fd, FALLOC_FL_PUNCH_HOLE | FALLOC_FL_KEEP_SIZE,
+                   block_off(*mbidv, mblocksz), mblocksz);
```

Comment:
The implementation of abort and delete are quite similar -- consider implementing a static function they can both use to avoid code duplication.